build.gradle.kts

```diff
@@ -337,6 +337,15 @@ tasks {
     systemProperty(
         "cody.autocomplete.enableFormatting",
         project.property("cody.autocomplete.enableFormatting") ?: "true")
+
+    val platformRuntimeVersion = project.findProperty("platformRuntimeVersion")
```

Comment:
I assume these changes allow us to specify the ide version as a runIde flag, is that right?

Reply:
Correct :)